src/components/modal/Modal.scss

```diff
@@ -144,25 +146,50 @@
             padding: 0;
         }
 
+        .modal-header-container {
+            position: absolute;
+            top: 0;
+            left: 0;
+            display: flex;
+            align-items: center;
+            width: 100%;
+            height: $responsive-modal-header-height;
+            background-color: $white;
+        }
+
+        .modal-header {
+            display: flex;
+            align-items: center;
+            padding: 0 16px;
+        }
+
         .modal-content {
-            padding-bottom: 60px;
+            margin-top: $responsive-modal-header-height;
+            margin-bottom: $responsive-modal-actions-height;
+            padding: 24px 16px 12px 16px;
+            overflow: auto;
         }
 
         .modal-dialog {
+            height: 100%;
             min-height: 100%;
```

Comment:
I think we can remove `min-height` now that we're specifying `height`.

Reply:
Removed `min-height`.